src/wiki/getting-started/install-of-alternatives.md

```diff
@@ -35,30 +47,46 @@ To install in Prism Launcher, create an instance with the **Fabric (or Quilt)**
 
-### <img src="https://cdn-raw.modrinth.com/data/BYfVnHa7/366754519a9702b43f0c9462b24de2757393efb9.png" height="20"> Simply Optimized
+### <img src="https://cdn-raw.modrinth.com/data/BYfVnHa7/7f8dc20fc0edd29fd95819a6f40938be0b9cadfa.png" height="20"> Simply Optimized
 
 [Simply Optimized](https://modrinth.com/modpack/sop) is a modpack designed with just optimization in mind. SO has better out-of-the-box performance than Fabulously Optimized, but it doesn't come with the QoL mods or full OptiFine parity you would see in Fabulously Optimized, so you're expected to add any additional mods you want yourself.
 
-## <img src="https://avatars0.githubusercontent.com/u/1390178?s=400&v=4" height="20"> Forge
+### <img src="https://raw.githubusercontent.com/skywardmc/art/main/adrenaline/logo_512h.png" height="20"> Adrenaline
+
+[Adrenaline](https://modrinth.com/modpack/adrenaline) is a modpack which aims to improve performance as much as possible while not changing anything about the vanilla game or adding any QOL features, and remaining stable. Adrenaline supports both Fabric and Quilt and a wide range of Minecraft versions. However, like Simply Optimized, it does not come with any OptiFine replacement mods, so you'll have to manually install the features that you want.
+
+### <img src="https://raw.githubusercontent.com/skywardmc/art/main/additive/logo_512h.png" height="20"> Additive
 
-### Note about Forge
+[Additive](https://modrinth.com/modpack/additive) is a modpack similar to Fabulously Optimized which aims to support nearly all OptiFine features. It's built from Adrenaline for best performance. Additive supports both Fabric and Quilt and a wide range of Minecraft versions.
+
+## <img src="https://avatars0.githubusercontent.com/u/1390178?s=400&v=4" height="30"> Forge Mods
 
 Unlike on Fabric/Quilt, most unofficial ports of OptiFine alternatives for Forge might have some compatibility/stability issues.
 
 While on some modpacks/with some mods they're going to work good, with others they might be *very* unstable so be well aware of what you're doing. If you have issues, use OptiFine.
 
+You can use [Sinytra Connector](https://modrinth.com/mod/connector) to run many Fabric mods on Forge.
+
 **Don't report issues with those on upstream's Discord support channels!**
 
+### <img src="https://raw.githubusercontent.com/embeddedt/embeddium/16.x/forge/src/main/resources/icon.png" height="20"> Embeddium
+
+[Embeddium](https://modrinth.com/mod/embeddium) is a Sodium port for Forge that focuses on compatibility with other Forge mods. We generally recommend using it instead of Rubidium. Be aware of the statements above before using it.
+
+To install in Prism Launcher, create an instance with the **Forge** loader, and then go to the *Mods tab*, click **Download Mods**. There, search for *Embeddium*, click on it, click **Select mod for download**, and then press **OK**.
```

Comment:
i don't think mod installation guidance is needed here. we already have the [installing mods](https://prismlauncher.org/wiki/getting-started/download-mods/) section on the wiki, which could be linked instead

edit: reading back i noticed there are other sections with this already, but imo they should be removed as well since it's pretty repetitive and duplicated information